modules/statics/src/coins.ts

```diff
@@ -739,7 +746,7 @@ export const coins = CoinMap.fromCoins([
   erc20('tlab', 'TLAB', 18, '0x36222730e78a8656230c54f9e402a062d168d2d9', UnderlyingAsset.TLAB),
   erc20('tnt', 'Tierion', 8, '0x08f5a9235b08173b7569f83645d2c7fb55e8ccd8', UnderlyingAsset.TNT),
   erc20('tok', 'Tokenplace', 8, '0x4fb721ef3bf99e0f2c193847afa296b9257d3c30', UnderlyingAsset.TOK),
-  erc20('trac', 'Trace', 18, '0xaa7a9ca87d3694b5755f213b5d04094b8d0f0a6f', UnderlyingAsset.TRAC),
+  erc20('trac', 'OriginTrails', 18, '0xaa7a9ca87d3694b5755f213b5d04094b8d0f0a6f', UnderlyingAsset.TRAC),
```

Comment:
Breaking change needs to be documented in the commit footer.

Reply:
Does this one need to change? The Token Tracker has it listed as Trace.

Reply:
was request in this ticket https://bitgoinc.atlassian.net/browse/BG-54049